editor/editor_config.cpp

```diff
@@ -120,9 +120,22 @@ EditorConfig::EditorConfig(string const & fileName)
   Reload();
 }
 
-bool EditorConfig::GetTypeDescription(vector<string> const & classificatorTypes,
+bool EditorConfig::GetTypeDescription(vector<string> && classificatorTypes,
```

Comment:
Чтобы убрать уже обработанный тип building, и не тратить время на ненужное сравнение строк внутри лямбды.

> On Apr 4, 2016, at 12:32, ygorshenin <notifications@github.com> wrote:
> 
> In editor/editor_config.cpp <https://github.com/mapsme/omim/pull/2670#discussion_r58349642>:
> 
> > @@ -120,9 +120,22 @@ EditorConfig::EditorConfig(string const & fileName)
> >    Reload();
> >  }
> >  
> > -bool EditorConfig::GetTypeDescription(vector<string> const & classificatorTypes,
> > +bool EditorConfig::GetTypeDescription(vector<string> && classificatorTypes,
> Если код вида GetTypeDescription(CreateVector()), то непонятно зачем ты делаешь здесь erase() из classificatorTypes.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2670/files/af10e69d522742081c4007f6da63a022705f438b#r58349642>